assembly/src/assembler/instruction/ext2_ops.rs

```diff
@@ -42,32 +42,38 @@ pub fn ext2_sub(span: &mut SpanBuilder) -> Result<Option<CodeBlock>, AssemblyErr
 /// b = (b0, b1) represent elements in the extension field of degree 2, this series of operations
 /// outputs the product c = (c1, c0) where c0 = a0b0 - 2(a1b1) and c1 = (a0 + a1)(b0 + b1) - a0b0
 ///
-/// This operation takes 24 VM cycles.
+/// This operation takes 3 VM cycles.
 pub fn ext2_mul(span: &mut SpanBuilder) -> Result<Option<CodeBlock>, AssemblyError> {
-    span.add_ops(ext2_mul_ops())
+    span.add_ops([Ext2Mul, Drop, Drop])
 }
 
 /// Given a stack in the following initial configuration [a1, a0, b1, b0, ...] where a = (a0, a1)
 /// and b = (b0, b1) represent elements in the extension field of degree 2, this series of
 /// operations outputs the result c = (c1, c0) where c1 = a1 / b1 and c0 = a0 / b0.
 ///
-/// This operation takes 59 VM cycles.
+/// This operation takes 17 VM cycles.
 pub fn ext2_div(span: &mut SpanBuilder) -> Result<Option<CodeBlock>, AssemblyError> {
```

Comment:
Looking at this now, I'm actually not sure this is correct:

1. The comment is incorrect: we are not computing `c1 = a1 / b1` and `c0 = a0 / b0`, we are computing `c = a * b^(-1)`.
2. Given how the stack is laid out, we actually should be computing `b / a`. I would lay out the stack as `[b1, b0, a1, a0, ...]` and then it would be `a / b` and it would be consistent with how we describe division in the base filed via `div` instruction.

Reply:
Thanks. Fixed.
Regarding 2, the order was also reversed for `ext2sub` instruction so I changed it to match this convention and changed the ordering of `a` and `b` in doc comments for all the ext2 instructions. Please let me know if you disagree with the change.
The order of `a` and `b` is different in the constraints so I've kept it the different than the instruction there.
So, for instructions, the stack is represented as `[b1, b0, a1, a0]`.
For the constraints, the stack is represented as `[a1, a0, b1, b0]`.

Reply:
Yeah - I think it is fine to leave as is.